gui/src/components/logs/FormattedLogMessage.svelte

```diff
@@ -68,26 +68,21 @@
   }
 
   @keyframes blink {
-    0% {
```

Comment:
Was this broken somehow? Did you try your changes in multiple browsers in Storybook? :)

Reply:
I figured it makes more sense to use `opacity` rather than `visibility` to avoid running into any unexpected behavior around the pointer events being removed for `visibility: hidden`, but since `opacity` has intermediate values between `0` and `1` it needs the `49% -> 50%` keyframes

Only tested in Chromium but will try others before merge. Opacity is very old and well supported though.

Reply:
So this is cartoonishly low priority, as blink is virtually unused and I added it just for fun, but...

I really think it made mode sense before. When you mess with opacity, you need to do alpha blending and stuff like that. There is some amount of time between 49% and 50%, so you're going to have some slight animation in there. It was binary, discrete, now it's scalar, continuous, even if there are sharp edges.

No need to change it back, but I don't think there was a change it in the first place.